src/common/hooks/useFieldErrorMsg.tsx

```diff
@@ -23,7 +31,7 @@ const useFieldErrorMsg = ({ name, inputError, validate }: Props): string | undef
     }
   }, [field.value, setHadValue]);
   const showError = field.value || (!field.value && hadValue);
-  const errorMessage = (showError ? error : '') || inputError;
+  const errorMessage = adaptErrorInvalidFormat((showError ? error : '') || inputError);
```

Comment:
if `error` is a falsy value then we'll display `inputError` despite `showError === true`. A bug?

Reply:
So, if there is no error from the validation side, it will display the error coming from `inputError` (from Formik directly).

I'm assuming those would be two different errors, and that the "inputError" needs to be shown if it's truthy.
https://github.com/openshift-assisted/assisted-ui-lib/blob/master/src/cim/components/Agent/BMCForm.tsx#L75